src/renderer/pages/BrowserPage/index.tsx

```diff
@@ -53,18 +55,49 @@ class BrowserPage extends React.PureComponent<Props> {
   }
 
   render() {
-    const { sleepy, disableBrowser, visible, partition } = this.props;
+    const {
+      sleepy,
+      disableBrowser,
+      visible,
+      partition,
+      lightMode,
+    } = this.props;
+
     if (sleepy && !visible) {
       return null;
     }
 
+    //Changes based on the bright mode checkbox
```

Comment:
Shoul be called `light` instead of `bright` for consistency

Reply:
That's fair.  Does keeping the "bright" nomenclature for user-facing purposes make sense though?  I found it to be probably the most reliably translatable way to communicate the functionality across the different languages that itch majorly supports.

Reply:
`Light mode` seems much more common as a term for this feature (e.g. Google search for `app light/bright mode` 😅 ). I can only speak for English though.